lib/bumblebee/text/token_classification.ex

```diff
@@ -169,6 +176,57 @@ defmodule Bumblebee.Text.TokenClassification do
     end)
   end
 
+  defp override_token_labels([entity | entities], spec, strategy) do
+    {final_group, word_entities} =
+      Enum.reduce(entities, {[entity], []}, fn entity, {word_group, word_entities} ->
+        if entity[:is_subword] do
```

Comment:
We want to assert the key is there, since we are adding it ourselves :)

```suggestion
        if entity.is_subword do
```